integration-tests/networks/known_networks.go

```diff
@@ -17,25 +18,6 @@ import (
 // Some networks with public RPC endpoints are already filled out, but make use of environment variables to use info like
 // private RPC endpoints and private keys.
 var (
-	// To create replica of simulated EVM network, with different chain ids
```

Comment:
Are these pruning off intentional? I added this to ensure changes are synced from other repos